ratatool-diffy/src/main/scala/com/spotify/ratatool/diffy/BigDiffy.scala

```diff
@@ -276,6 +276,13 @@ object BigDiffy extends Command {
                    diffy: TableRowDiffy): BigDiffy[TableRow] =
     diff(sc.bigQueryTable(lhs), sc.bigQueryTable(rhs), diffy, keyFn)
 
+  /** Diff two SCollection[T] **/
+  def diffCaseClass[T : ClassTag : MapEncoder](lhs: SCollection[T],
```

Comment:
Can we move this and the shapeless deps all into `ratatoolShapeless` Would prefer not to force users to pull in Shapeless unless they need it.